lighthouse-core/gather/gatherers/fonts.js

```diff
@@ -138,19 +151,28 @@ function getFontFaceFromStylesheets() {
   const corsDataPromises = [];
   // Get all loaded stylesheets
   for (const stylesheet of Array.from(document.styleSheets)) {
+    const cssStylesheet = /** @type {CSSStyleSheet} */ (stylesheet);
+
     try {
-      const cssStylesheet = /** @type {CSSStyleSheet} */ (stylesheet);
-      // Cross-origin stylesheets don't expose cssRules by default. We reload them w/ CORS headers.
-      if (cssStylesheet.cssRules === null && cssStylesheet.href && cssStylesheet.ownerNode &&
-        // @ts-ignore - crossOrigin exists if ownerNode is an HTMLLinkElement
-        !cssStylesheet.ownerNode.crossOrigin) {
+      // cssRules can be null or this access can throw when CORS isn't enabled, throw a matching error message.
```

Comment:
a little hard to parse comment. Semicolon instead of a comma?

Reply:
done